sdk/spring/spring-cloud-azure-stream-binder-eventhubs/README.md

```diff
@@ -137,15 +167,20 @@ The mode in which checkpoints are updated.
 
 `RECORD`, checkpoints occur after each record is successfully processed by user-defined message handler without any exception. If you use `StorageAccount` as checkpoint store, this might become botterneck. 
 
-`BATCH`, checkpoints occur after each batch of messages successfully processed by user-defined message handler without any exception. `default` mode. You may experience reprocessing at most one batch of messages when message processing fails. Be aware that batch size could be any value.
+`BATCH`, checkpoints occur after each batch of messages successfully processed by user-defined message handler 
+without any exception. `default` mode. You may experience reprocessing at most one batch of messages when message 
```

Comment:
BATCH is not the default mode, while RECORD is. Please update the doc .

Reply:
"You may experience reprocessing at most one batch of messages when message processing fails" is this applied?

Reply:
updated